src/ui/pages/HomePage/HomePage.styles.ts

```diff
@@ -0,0 +1,16 @@
+import { BgTextures, UiTextures } from "../../../App.types";
+import { Point, Polygon } from "pixi.js";
+
+export const HomePageStyles = {
```

Comment:
You could define types to this styles.  It is not strict type right now.

Reply:
updated as suggested;

https://github.com/irgat/soloTest/pull/3/commits/59873d9858f51cfb004a4396a5f2b8e0464ada68